tycho-plugins/repository-utils/src/main/java/org/jboss/tools/tycho/sitegenerator/FetchSourcesFromManifests.java

```diff
@@ -129,6 +140,9 @@
 	@Parameter(property = "fetch-sources-from-manifests.outputFolder", defaultValue = "${basedir}/zips")
 	private File outputFolder;
 
+	@Parameter(property = "fetch-sources-from-manifests.sourcesZip", defaultValue = "${project.build.directory}/fullSite/all/jbosstools-src.zip")
```

Comment:
No, we're merging 17 zips into a single one used by jbosstools. And it's the src.zip, so therefore jbosstools-src.zip seems legit for the output filename.